app/controllers/report_controller/reports/editor.rb

```diff
@@ -810,7 +810,7 @@ def move_cols_right
         if params[:available_fields].include?(af[1])        # See if this column was selected to move
           unless @edit[:new][:fields].include?(af)          # Only move if it's not there already
             @edit[:new][:fields].push(af)                     # Add it to the new fields list
-            if af[0].include?(":")                            # Not a base column
+            if af[0].include?(":") && !af[1].include?(CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX) # Not a base column
```

Comment:
@simon3z  yes but this is ok for now,( we are doing do clean up about fields - this will be covered by any class of child MiqExpression::Target )